opencsp/common/lib/camera/ImageAcquisitionAbstract.py

```diff
@@ -84,62 +87,62 @@ def _check_saturated(im):
         if not _check_exposure_set():
             raise ValueError('Error with setting exposure.')
 
-        # Set final exposure and print results
+        # Set final exposure and log results
         exposure_value_set = exposure_values[_get_exposure_idxs()[0]]
         self.exposure_time = exposure_value_set
-        print(f'Exposure set to: {exposure_value_set}')
+        lt.info(f'Exposure set to: {exposure_value_set}')
 
     @abstractmethod
     def get_frame(self):
         """Gets a single frame from the camera"""
-        pass
 
-    @abstractproperty
+    @property
+    @abstractmethod

```

Comment:
I used `@abstractproperty` when I originally made this class a while ago. I recently found out is deprecated and should be replaced with 
```
@property
@abstractmethod
``` 
Sorry for the spam.

Reply:
You're good! I also had this realization recently.